### ISSUE TITLE
Phylactery damage instance applying before the spell damage, is it intended?

### ISSUE THREAD

**Issue by Magn0dota** [ValveSoftware/Dota2-Gameplay#8429]
### Item name

Phylactery

### Description

The separate 125 damage instance seems to be hitting before the actual spell's damage, when it hits Nyx's Carapace they take little to no damage and Nyx takes the full spell, if it's intended then ok, sucks to be a Nyx player but I don't think it is.

### Example Match ID and Timestamp

7120753396

**Comment by pontuslaestadius** [comment]
![image](https://user-images.githubusercontent.com/31597765/233824148-be456d32-49d9-446f-9344-c6eda511e950.png)

I don't think it says anywhere if the extra damage should be applied first or second, but it will always apply on the same tick as the original damage, even if it has a slow projectile speed.

**Comment by Magn0dota** [comment]
Yeah, it's not specified so I assume it shouldn't because for Phylactery to activate the spell needs to hit, and when it hits it should apply the spell damage immediately, there should be no room for the Phylactery damage to go in between these steps. At least that's how it would make sense to me.
